### ISSUE TITLE
Refund Bitcoin Cash

### ISSUE THREAD

**Issue by jmcgrath207** [schancel/cashaddr-converter#19]
Hi Schancel,

It appears that I send you bitcoin cash to your address `qpm2qsznhks23z7629mms6s4cwef74vcwvy22gdx6a` accidentally while using your app. For some reason when you add your address you have to click convert again once your switch tabs to legacy.

https://explorer.bitcoin.com/bch/tx/c461ec6cad0688bc79ba1e88baaff40373dfd13d210166fd2d0864bd95f4f91f

Could you send it to address `qza9vj66gvrnc4hehu8kpqqhgg0pzj8qnc8q5029sr` . You can keep 10% of the value for doing this favor.

**Comment by schancel** [comment]
> For some reason when you add your address you have to click convert again once your switch tabs to legacy.

Not sure what you mean here.  I use it all the time and haven't ran into this.

> It appears that I send you bitcoin cash to your address qpm2qsznhks23z7629mms6s4cwef74vcwvy22gdx6a accidentally while using your app

This is not the address that is filled in as an example.  Where did you get it at?

**Comment by jmcgrath207** [comment]
the address `qpm2qsznhks23z7629mms6s4cwef74vcwvy22gdx6a` is mentioned in your repo twice.

https://github.com/schancel/cashaddr-converter/search?q=qpm2qsznhks23z7629mms6s4cwef74vcwvy22gdx6a&unscoped_q=qpm2qsznhks23z7629mms6s4cwef74vcwvy22gdx6a

I think it's pretty remarkable that you generated a 42 length string randomly twice that happens to be the same and also a valid bitcoin cash address wallet, don't you think?

Look, I am assuming that is an error in your program and that this was an honest mistake. But claiming no knowledge of that address is just plain wrong.

**Comment by schancel** [comment]
These were part of the tests for the cashaddr code in Bitcoin-ABC.  They were generated by @dagurval as far as I know.  I copied and pasted them into this codebase when writing it.

How did you find these to accidentally send funds to?

**Comment by jmcgrath207** [comment]
Thanks for looking into this.

Here are the steps I have taken.

I went to https://cashaddr.bitcoincash.org/
I added my address and clicked the legacy tab.
It is here it gave me the address legacy address for `qpm2qsznhks23z7629mms6s4cwef74vcwvy22gdx6a`

After I realized the mistake, I clicked top right where it sent me to this repo page which is where I found the address in the source code.

I've tried recreating the test Scenario but it doesn't work.

Reviewing the WHOis record for this site, there is no information. 
https://whois.icann.org/en/lookup?name=cashaddr.bitcoincash.org

Know what I know now, I should have used the https://explorer.bitcoin.com/bch site since it converts the addresses as soon as you input them.
